DEVELOPER.md

```diff
@@ -363,9 +363,13 @@ You can manually fill in the versions, or follow the steps below.
 # cd kong
 
 export RESTY_VERSION=$(grep -oP 'RESTY_VERSION=\K.*' .requirements)
-export RESTY_OPENSSL_VERSION=$(grep -oP 'RESTY_OPENSSL_VERSION=\K.*' .requirements)
```

Comment:
should we just remove this “build from source” section, and put a reference link to kong-build-tools